manifests/config/audit_profiles/custom.pp

```diff
@@ -5,23 +5,19 @@
 # @example Passing an Array of Rules
 #
 #  ---
-#  auditd::config::audit_profiles::user_specified:
+#  auditd::config::audit_profiles::custom::rules:
 #    - "-a always,exit -F arch=b64 -S creat -F exit=-EACCES -k unsuccessful_file_operations"
 #    - "-w /etc/passwd -p wa -k passwd_changes"
 #
 # @example Passing an EPP Template
 #
 #  ---
-#  auditd::config::audit_profiles::user_specified:
-#    - "my_templates_module/auditd/my_audit_rules.epp"
-#    - "This will be ignored"
+#  auditd::config::audit_profiles::custom::template: "my_templates_module/auditd/my_audit_rules.epp"
 #
 # @example Passing an ERB Template
 #
 #  ---
-#  auditd::config::audit_profiles::user_specified:
-#    - "my_templates_module/auditd/my_audit_rules.erb"
-#    - "This will be ignored"
+#  auditd::config::audit_profiles::custom::template: "my_templates_module/auditd/my_audit_rules.erb"
```

Comment:
The 'ignored' lines should be retained since it's important for people to understand that EPP and ERB files take precedence. (unless that didn't work)

Reply:
Problem is, the [`$template`](https://github.com/simp/pupmod-simp-auditd/blob/master/manifests/config/audit_profiles/custom.pp#L41) is a `String`, not an `Array[String]`
```
  Optional[String[1]]        $template = undef
```

And non-conformant values are not ignored; they raise an [error](https://github.com/simp/pupmod-simp-auditd/blob/master/manifests/config/audit_profiles/custom.pp#L65):
```
      fail('Your template must end with either ".epp" or ".erb"')
```